department-of-veterans-affairs/caseflow
# Queue | Pop multiple breadcrumbs on jump backwards

Resolves an issue [pointed out in #appeals-queue](https://dsva.slack.com/files/U2P552W04/FA156PDS4/screen_shot_2018-04-05_at_11.25.01_am.png)

### Description
Previously, when jumping between pages via the breadcrumbs, we would never remove any crumbs, resulting in duplication. Now, if the page we are jumping to has a crumb that is rendered, we remove all crumbs after that one, effectively jumping backwards.

### Acceptance Criteria 
- [ ] Verify breadcrumbs are removed correctly on jumping to earlier pages

### Testing Plan
1. Start the draft decision checkout flow, get to final step
2. Click on a breadcrumb for the previous step, the Submit breadcrumb should be removed.
